routes/index.js

```diff
@@ -50,42 +53,59 @@ router.post('/api/users', function (req, res) {
     console.log('Input field empty')
     return res.status(400).send('Input field empty')
   }
-
   // Checks for duplicate
-  const checkdup = {
-    text: 'SELECT COUNT(email) FROM users WHERE email = $1',
-    values: [email.toLowerCase()]
-  }
-  db.query(checkdup)
+  CheckDup(email)
     .then(result => {
-      if (result.rows[0].count !== 0) {
-        res.status(240)
+      if (result != 0) {
+        // console.log(result)
+        res.status(409).send({message:"exist"})
+      } else {
```

Comment:
We can reduce the amount of indentation here (and make it easier to read) by using `await`, just like you did in the `CheckDup` function itself. 

This is true because we're not doing anything in parallel with the `checkdup` call; thus, we should just make it "serial" (wait for the first line to complete, then move to the next). If we were to have code that would execute after the `CheckDup`, regardless of the outcome, then only then does the `.then` paradigm help.

Reply:
Since this isn't part of an async function, does using await still work?